lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -125,9 +125,11 @@ sub create_from_settings {
     die 'The ' . join(',', @invalid_keys) . " cannot include / in value\n" if @invalid_keys;
 
     # validate special settings
-    my %special_settings = (_PRIORITY => delete $settings{_PRIORITY});
+    my %special_settings = (TEST => delete $settings{TEST}, _PRIORITY => delete $settings{_PRIORITY});
     my $validator = Mojolicious::Validator->new;
+    $validator->add_check(no_newline => sub ($v, $name, $value, @args) { index($value, "\n") != -1 });
     my $v = Mojolicious::Validator::Validation->new(validator => $validator, input => \%special_settings);
+    my $test = $v->required('TEST')->like(TEST_NAME_REGEX)->no_newline->param;
```

Comment:
That shouldn't be necessary, see https://github.com/os-autoinst/openQA/pull/6195/files#r1965526523

Reply:
Thanks, I'll change the code.